avocado/utils/service.py

```diff
@@ -148,39 +148,38 @@ def method(cmd_result):
             if cmd_result.exit_status:
                 raise process.CmdError(cmd_result.command, cmd_result)
             # The final dict to return.
-            _service2statusOnTarget_dict = {}
+            services_statuses_dict = {}
```

Comment:
What I miss here is the specific pylint fixes that are being applied.  And, if it all possible, we should attempt to fix them in Avocado as a whole, and enable it according to https://trello.com/c/Ohh7Q6qj/1183-enable-most-warning-checks-in-travis-make-check

Reply:
I mean to do something like https://github.com/avocado-framework/aexpect/pull/41 for Avocado as well, this was just reaction to poor state of this module. Instead of going for fix this error everywhere I simply fixed all (addressable) pylint issues for this file (and kept the intrusive for next commit).